### PR TITLE
Update release notes draft to version v1.22.0-alpha.2

### PR DESCRIPTION
**Pull request by Damans227**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.22.0-alpha.2

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Comment by Damans227** [comment]
Putting it on hold for now. 
/hold

**Comment by puerco** [comment]
OK, it looks good to me now. Thanks for regenerating it @Damans227 could you open an issue about maps not getting pulled when running without `--fix`?

**Comment by Damans227** [comment]
> OK, it looks good to me now. Thanks for regenerating it @Damans227 could you open an issue about maps not getting pulled when running without `--fix`?

Sure, here it is: https://github.com/kubernetes/release/issues/2079

File: releases/release-1.22/release-notes/maps/pr-100731-map.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pr: 100731
+releasenote:
+  text: 'azurefile: Normalize share name to not include the capital letters'
```
Comment (sladyn98) [inline releases/release-1.22/release-notes/maps/pr-100731-map.yaml:3]: Any reason why this is single quotes ? Just curious

File: releases/release-1.22/release-notes/maps/pr-101685-map.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pr: 101685
+releasenote:
+  text: Intree volume plugin scaleIO support is been completely removed from Kubernetes.
```
Comment (sladyn98) [inline releases/release-1.22/release-notes/maps/pr-101685-map.yaml:3]:
```suggestion
  text: Intree volume plugin scaleIO support has been completely removed from Kubernetes.
```

File: releases/release-1.22/release-notes/maps/pr-101938-map.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pr: 101938
+releasenote:
+  text: 'kubeadm: Remove the deprecated command `kubeadm alpha kubeconfi`. Please
```
Comment (sladyn98) [inline releases/release-1.22/release-notes/maps/pr-101938-map.yaml:3]:
```suggestion
  text: 'kubeadm: Remove the deprecated command `kubeadm alpha kubeconfig`. Please
```

File: releases/release-1.22/release-notes/maps/pr-100731-map.yaml (hunk 1 above)
Comment (Damans227) [reply releases/release-1.22/release-notes/maps/pr-100731-map.yaml:3]: > Any reason why this is single quotes ? Just curious

Hmm, not really sure and they don't make any sense too. Do you think we should remove them ?

Comment (pmmalinov01) [reply releases/release-1.22/release-notes/maps/pr-100731-map.yaml:3]: You need them because YAML will complain about having `:`

File: releases/release-1.22/release-notes/maps/pr-101496-map.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+pr: 101496
+releasenote:
+  text: Pod Affinity `NamespaceSelector` and the associated `CrossNamespaceAffinity`
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-101496-map.yaml:3]: `podAffinity` is a key from the spec, may it should be one word and code blocked. Also when we have a feature or something like it, that moves to beta we try to phrase it as:
**FOO FAR BAR feature graduates to Beta and enabled by default.**
Can you try to rephrase it to fit the above?

File: releases/release-1.22/release-notes/maps/pr-101542-map.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+pr: 101542
+releasenote:
+  text: Scheduler's CycleState now embeds internal read/write locking inside its Read()
+    and Write() functions. Meanwhile, Lock() and Unlock() function are removed.                 scheduler
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-101542-map.yaml:4]: may Write(), Read(), Lock() and  Unlock() should be in code blocks as they are methods. Also, there is a lot of white spaces and Scheduler should be with a capital letter.

File: releases/release-1.22/release-notes/maps/pr-101560-map.yaml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+pr: 101560
+releasenote:
+  do_not_publish: true
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-101560-map.yaml:3]: I do think this PR needs to be published and to have a proper release note. I see the owner did not set one, that does not mean we should not try to investigate and set a good explanation. 
Example:

Fixes a panic in `kube-scheduler` when accessing not existing pods while using `DefaultPreemption ` plugin.

File: releases/release-1.22/release-notes/maps/pr-102030-map.yaml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+pr: 102030
+releasenote:
+  text: The ingress v1beta1 has been deprecated.
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-102030-map.yaml:3]: `v1beta1`

File: releases/release-1.22/release-notes/maps/pr-102108-map.yaml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+pr: 102108
+releasenote:
+  text: 'kubead: remove the deprecated `--csr-only` and `--csr-dir` flags from `kubeadm
+    #     init phase certs`. Deprecate the same flags under `kubeadm certs renew`.
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-102108-map.yaml:4]: Spacing is a bit off also you must remove the `#` from this and below line.

**Comment by pmmalinov01** [comment]
@Damans227 left a couple of points, great work so far.

File: releases/release-1.22/release-notes/maps/pr-101496-map.yaml (hunk 4 above)
Comment (Damans227) [reply releases/release-1.22/release-notes/maps/pr-101496-map.yaml]: > `podAffinity` is a key from the spec, may it should be one word and code blocked. Also when we have a feature or something like it, that moves to beta we try to phrase it as:
> **FOO FAR BAR feature graduates to Beta and enabled by default.**
> Can you try to rephrase it to fit the above?

Done.

File: releases/release-1.22/release-notes/maps/pr-102030-map.yaml (hunk 7 above)
Comment (Damans227) [reply releases/release-1.22/release-notes/maps/pr-102030-map.yaml]: > `v1beta1`

Done

File: releases/release-1.22/release-notes/maps/pr-101542-map.yaml (hunk 5 above)
Comment (Damans227) [reply releases/release-1.22/release-notes/maps/pr-101542-map.yaml]: Done

File: releases/release-1.22/release-notes/maps/pr-102108-map.yaml (hunk 8 above)
Comment (Damans227) [reply releases/release-1.22/release-notes/maps/pr-102108-map.yaml]: Done

File: releases/release-1.22/release-notes/maps/pr-101560-map.yaml (hunk 6 above)
Comment (Damans227) [reply releases/release-1.22/release-notes/maps/pr-101560-map.yaml]: Done

**Comment by Damans227** [comment]
> /lgtm

Yay!!! :)
